mne/datasets/sleep_physionet/age.py

```diff
@@ -94,7 +104,28 @@ def fetch_data(subjects, recording=[1, 2], path=None, force_update=False,
     path = data_path(path=path, update_path=update_path)
     params = [path, force_update, base_url]
 
-    _check_subjects(subjects, 20)
+    _check_subjects(
+        subjects, 83, missing=[39, 68, 69, 78, 79], on_missing=on_missing)
+
+    # Check for missing recordings
+    if set(subjects) & {36, 52} and 1 in recording:
+        msg = ('Requested recording 1 for subject 36 and/or 52, but it is not '
+               'available in corpus.')
+        if on_missing == 'error':
+            raise ValueError(msg)
+        elif on_missing == 'warning':
+            warn(msg)
+        else:
+            pass
+    if 13 in subjects and 2 in recording:
+        msg = ('Requested recording 2 for subject 13, but it is not available '
+               'in corpus.')
+        if on_missing == 'error':
+            raise ValueError(msg)
+        elif on_missing == 'warning':
+            warn(msg)
+        else:
+            pass
```

Comment:
please do refactor to a `_on_missing(on_missing, msg)`. It looks like there are instances in forward, epochs, and montage that can be refactored to use the same pattern. I'd prefer to standardize around NumPy's naming (at least the subset we use) as raise/warn/ignore, at least in the docstring (the code can treat raise/error and warn/warning equivalently for painless backward compat).

Reply:
Where would you put `_on_missing`? Would `utils/check.py` be the right place?